src/xdocs/config_naming.xml

```diff
@@ -488,10 +492,14 @@ class FourthClass {} // OK
         </source>
         <p>Example:</p>
         <source>
-abstract class AbstractFirstClass {} // OK
-abstract class SecondClass {} // violation, it should match pattern "^Abstract.+$"
-class AbstractThirdClass {} // OK, no "abstract" modifier
```

Comment:
We are losing some example context. `OK`s were completely removed.

Reply:
ok's were requested to be removed at https://github.com/checkstyle/checkstyle/pull/13112#discussion_r1212393229

Reply:
Wasn't he just responding for exactly "Ok" as a comment? This highlighted comment has more wording and can provide some extra context imo.

Reply:
After hundreds of example PR, I  , and not only I found that comment `ok` is just a noise that makes it hard to read.
This example is small, so you will not fill pain, but, there are more complicated examples and it is not good. 
I agree to put comment to explain some not obvious reason, but it will be exception.

Reply:
Then it is my understand we are going to restore these "OK, blah blah blah"

Reply:
For now I treat all `//ok` comments as stuff to remove, there might be cases to keep them as form of  `OK, blah blah blah` if it is not obvious.
But in this check they are not required.

Reply:
Still seeing this as something to be restored in input file.

Reply:
We do this removals even in test inputs. 
Same is here it just makes example full of content that is hard to distinguish without code coloring.
All user should see is comment as red flag of attention. 
All other lines are without attention, as no violation on them, on all of them.

We use to have such comments only at times when we had no enforcing and it was very manual markers , but it lost its role after we finished Input based tests

Reply:
Such comment become to appear in completely unrelated places, by copy paste, and keep it right place is not possible. Better without it .

Reply:
The only place where we kept ok comment as good is at Indentation. But to be honest, even here it is noise but we keep it as 'ok' is not whole content of comment, it is trailing part of it, so absence of it doesn't play much role as comment is for indentation values. So we abandoned it even here.

https://github.com/checkstyle/checkstyle/blob/c5566a6ea8c336ddedde446648b03ef56a73b7ba/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationAnnotationScopeIndentationCheck.java#L37

Reply:
The only good usage of "ok" with extra wording if that is not obvious, I can not find such cases from phone, but they exist, or could come up at any time.

I think we did them in check for final modifier , as some items are very very not obvious or limitations of Check.

Limitation highlight is also good reason to use "ok"

Reply:
I created issue to remove almost all 'ok' comments.
https://github.com/checkstyle/checkstyle/issues/13213